Description: foomatic-rip: SECURITY FIX: Also consider the semicolon (';') as an
 illegal shell escape character.
Author: Till Kamppeter <till.kamppeter@gmail.com>
Origin: upstream, https://bzr.linuxfoundation.org/loggerhead/openprinting/cups-filters/revision/7419
Bug-Debian: https://bugs.debian.org/807930

--- a/filter/foomatic-rip/util.c	2015-10-30 15:45:03 +0000
+++ b/filter/foomatic-rip/util.c	2015-12-12 23:27:21 +0000
@@ -31,7 +31,7 @@
 #include <assert.h>
 
 
-const char* shellescapes = "|<>&!$\'\"`#*?()[]{}";
+const char* shellescapes = "|;<>&!$\'\"`#*?()[]{}";
 
 const char * temp_dir()
 {
